test/core/unreached-invalid.wast

```diff
@@ -249,6 +249,42 @@
   ))
   "type mismatch"
 )
+(assert_invalid
+  (module (func $type-unary-num-vs-void-in-block-after-unreachable
+    (unreachable) (block (drop (i32.eq (nop))))
+  ))
+  "type mismatch"
+)
+(assert_invalid
+  (module (func $type-unary-num-vs-void-in-loop-after-unreachable
+    (unreachable) (loop (drop (i32.eq (nop))))
```

Comment:
`eqz`?

Reply:
Done, and moved the loop tests up after line 197.